### ISSUE TITLE
Setting language is a little confusing (terminal tab focus mode)

### ISSUE THREAD

**Issue by sbatten** [microsoft/vscode#175053]
Testing #174707

The override language makes me think that terminal setting inherits from the editor setting if unset, but they seem like totally independent settings.

**Comment by meganrogge** [comment]
I agree, which is why I didn't want to mention the other setting in the first place. @Tyriar thought it was important to do because historically the terminal and editor have shared this state.

**Comment by meganrogge** [comment]
I'm removing the mention of the other one as we call these out in each area's help menu

**Comment by Tyriar** [comment]
> The override language makes me think that terminal setting inherits from the editor setting if unset, but they seem like totally independent settings.

We probably should be inheriting if it's unset (`true | false | null`?), otherwise it could confuse the user why tab isn't moving focus only on the terminal. The link to the other setting was meant to give more context for the setting as it interacts directly with it.

**Comment by meganrogge** [comment]
I think it will be very confusing if the terminal inherits from the editor setting here. I know we do that with font, but I've never been a fan of that approach. 

It makes sense IMO for the setting to be `off` by default for both
